teams/rustdoc-contributors.toml

```diff
@@ -0,0 +1,23 @@
+name = "rustdoc-contributors"
+subteam-of = "rustdoc"
+
+[people]
+leads = []
+members = [
+    "fmease"
+]
+
+[permissions]
+crater = true
```

Comment:
Are we sure this is necessary?

Reply:
We can remove `crater` permissions for the time being. Can always be added later on or rustdoc team members can run it if it seems necessary.

Reply:
Done.